src/js/components/NavBar.js

```diff
@@ -84,7 +97,7 @@ class NavBar extends Component {
       goToChapters = () => {
         const {getChapters} = this.props;
         getChapters(searchBar.projectId);
-        history.push(`/chapters?projectId=${searchBar.projectId}&&bookName=${searchBar.bookName}&&mode=${searchBar.mode}`);
+        history.push(`/chapters?projectId=${searchBar.projectId}&bookName=${searchBar.bookName}&mode=${searchBar.mode}`);

```

Comment:
and can you reformat this the same way you did the other `history.push` in this PR?